nix/t1/default.nix

```diff
@@ -38,29 +38,29 @@ lib.makeScope newScope
       ip = {
         recurseForDerivations = true;
 
-        mlirbc = innerSelf.callPackage ./mlirbc.nix { target = "ip"; };
+        mlirbc = innerSelf.callPackage ./mlirbc.nix { target = "ip"; use-binder = true; };
```

Comment:
use-binder is a boolean value,  I recommend using "useBinder" instead of "use-binder", kebab-case is often used as package name.